Edit: Plans and Features (#244)
ritza-co/fusionauth-site: astro/src/content/docs/get-started/core-concepts/plans-features.mdx

## Self-Hosted FusionAuth Versus Cloud-Hosted FusionAuth

> Once you have chosen a FusionAuth plan with the features you want, you need to decide where to host FusionAuth. You can either host it with the FusionAuth company, Inversoft, or you can host it yourself, whether on-premise or with a cloud service provider like Hetzner or AWS. Your decision to self-host or use Inversoft is independent of which FusionAuth plan you choose. Any FusionAuth plan can be hosted either by you or by us. There is no difference in plan features between hosts.

Author: ```suggestion
Once you have chosen a FusionAuth plan with the features you want, you need to decide where to host your FusionAuth instance. You can host it yourself, either on-premise server or with a cloud service provider like Hetzner or AWS, or you can use FusionAuth Cloud to host your instance. Your decision to self-host or register for FusionAuth Cloud is independent of which plan you choose. There are no differences in plan features between hosts.
```
